mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java

```diff
@@ -147,27 +142,33 @@ public void service(Mail mail) throws MessagingException {
     private Stream<Pair<String, byte[]>> toJson(Map.Entry<String, Calendar> entry, Mail mail) {
         return mail.getRecipients()
             .stream()
-            .flatMap(recipient -> toJson(entry.getValue(), recipient, mail.getSender(), mail.getName()))
+            .flatMap(recipient -> toICAL(entry.getValue(), recipient, mail.getSender()))
+            .flatMap(ical -> toJson(ical, mail.getName()))
```

Comment:
We use overwrite method then sometime it makes me confuse that we call recurrence